manifests/n/Nirsoft/UninstallView/1.51/Nirsoft.UninstallView.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Nirsoft.UninstallView

```

```suggestion
PackageIdentifier: NirSoft.UninstallView
```
Please update file name and path to match this change.